lib/webmention.rb

```diff
@@ -1,8 +1,27 @@
-require 'httparty'
+require 'absolutely'
+require 'addressable/uri'
+require 'http'
+require 'indieweb/endpoints'
 require 'nokogiri'
-require 'open-uri'
-require 'set'
 
 require 'webmention/version'
+require 'webmention/exceptions'
+
 require 'webmention/client'
-require 'webmention/endpoint'
+require 'webmention/http_request'
+require 'webmention/registerable'
+
+require 'webmention/parsers'
+require 'webmention/parsers/html_parser'
+
+module Webmention
+  class << self
+    def client(source)
+      Client.new(source)
+    end
+
+    def send_mention(source, target)
+      client(source).send_mention(target)
+    end
+  end
```

Comment:
As noted in the `README`, these are convenience methods for quickly using the gem:

```rb
Webmention.send_mention('https://source.example.com/100', 'https://target.example.com/100')
```